setup.cfg

```diff
@@ -0,0 +1,61 @@
+[metadata]
+name = Twisted
+version = attr: src.twisted.__version__
+description = An asynchronous networking framework written in Python
+author = Twisted Matrix Laboratories
+author_email = twisted-python@twistedmatrix.com
+maintainer = Glyph Lefkowitz
+maintainer_email = glyph@twistedmatrix.com
+url = https://twistedmatrix.com/
+project_urls =
+    Documentation=https://twistedmatrix.com/documents/current/
+    Source=https://github.com/twisted/twisted
+    Issues=https://twistedmatrix.com/trac/report
+license = MIT
+classifiers =
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
```

Comment:
3.9 was released this week and CI passes on 3.9-rc2, so I think it's safe to add 3.9 to the classifiers as well.

Reply:
```suggestion
    Programming Language :: Python :: 3.8
    Programming Language :: Python :: 3.9
```